docs/calico-with-docker/AWS.md

```diff
@@ -173,13 +173,15 @@ aws ec2 run-instances \
   --key-name mykey \
   --security-group-ids $SECURITY_GROUP_ID \
   --user-data file://<PATH_TO_CLOUD_CONFIG>/user-data-first
-#  --subnet $SUBNET_ID 
+#  --subnet $SUBNET_ID
 #  Include the subnet param above if using a non-default VPC
+
+INSTANCE_ID_1=<InstanceId>
```

Comment:
There should be a note either be a comment added above here to tell the used to copy the instance ID into this command, or a line above the command to also save the returned instance-id from the output.

This goes for `INSTANCE_ID_2` below also.